core/src/main/java/org/jboss/intersmash/IntersmashConfig.java

```diff
@@ -407,4 +414,20 @@ public static String keycloakOperatorChannel() {
 	public static String keycloakOperatorPackageManifest() {
 		return XTFConfig.get(KEYCLOAK_OPERATOR_PACKAGE_MANIFEST, DEFAULT_KEYCLOAK_OPERATOR_PACKAGE_MANIFEST);
 	}
+
+	public static String openDataHubOperatorCatalogSource() {
```

Comment:
those methods don't look like getters. I understand that other methods here in this class doesn't have javadoc, but does it make sense to add javadoc to new methods here?

Reply:
I do agree. Maybe we can add real getters in the future, and turn the current ones into alias, and deprecate them.